wpilib/wpilib/notifier.py

```diff
@@ -0,0 +1,124 @@
+# validated: 2017-12-24 DS 8b7aa61091df edu/wpi/first/wpilibj/Notifier.java
+#----------------------------------------------------------------------------
+# Copyright (c) FIRST 2016-2017. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+#----------------------------------------------------------------------------
+
+import threading
```

Comment:
Since the rio has 2 cores, would it be beneficial to try and use multiprocessing instead of threading? I don't know much about it, so idk how hard it'd be to implement, but from my reading it seems that multiprocesses side-step the GIL

Reply:
No. You would break the HAL, since you then have the HAL loaded in 2 processes.

Reply:
Is that generally a bad thing? Why can you have it in two threads but not two processes?

Reply:
It was never built to be distributed across processes, so you can't.

Reply:
Oh cause you're modifying data and such?